app/src/main/java/com/practicum/playlistmaker/Track.kt

```diff
@@ -0,0 +1,8 @@
+package com.practicum.playlistmaker
+
+data class Track(
+    val trackName: String,
+    val artistName: String,
+    val trackTime: String,
+    val artworkUrl100: String
```

Comment:
```suggestion
    val artworkUrl100: String?
```
Картинка теоретически может не прийти